scripts/log-capture

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+date=$(date +%F_%R)
+OUTDIR=/tmp/LogCapture-${date}
+ARCHIVE=${1-/tmp/log-capture.tar.bz2}
+
+_to_log() { local OutputFile=$(tr ' /' '_' <<<"$@"); $@ >${OUTDIR}/${OutputFile}; }
+
+mkdir -p ${OUTDIR}
+cd ${OUTDIR}
+
+if [[ ! -f /tmp/ks.cfg ]];then
+  echo "No /tmp/ks.cfg present" > ${OUTDIR}/ks.cfg
+fi
+
+# the /mnt/sysimage/root/anaconda-ks.cfg file
+# might not yet exist when this script is called
+if [[ ! -f  /mnt/sysimage/root/anaconda-ks.cfg ]];then
+    echo "No /mnt/sysimage/root/anaconda-ks.cfg" > ${OUTDIR}/ks.cfg
```

Comment:
Actually, shouldn't the output be anaconda-ks.cfg ? Eq. two separate files ? Also I think we should include /tmp/ks.cfg in the tarball as well.